src/TraceEvent/Symbols/SymbolReader.cs

```diff
@@ -229,6 +231,11 @@ public string FindSymbolFilePath(string pdbFileName, Guid pdbIndexGuid, int pdbI
                         }
 
                         pdbPath = GetFileFromServer(element.Target, pdbIndexPath, Path.Combine(cache, pdbIndexPath));
+
+                        if (pdbPath == null && portablePdbMatch)
+                        {
+                            pdbPath = GetFileFromServer(element.Target, pdbSimpleName + @"\" + pdbIndexGuid.ToString("N").ToUpper() + "FFFFFFFF" + @"\" + pdbSimpleName, Path.Combine(cache, pdbIndexPath));
```

Comment:
Can you please add a comment here on what the concatenated string path should look like here?